packages/frontend/src/components/ProjectAccess/ProjectAccess.tsx

```diff
@@ -139,7 +140,8 @@ const ProjectAccess: FC<ProjectAccessProps> = ({
                         {
                             type: 'organization',
                             role: convertOrganizationRoleToProjectRole(
-                                orgRoles[orgUser.userUuid],
+                                orgRoles[orgUser.userUuid] ||
```

Comment:
this was the main fix, if `orgRoles[orgUser.userUuid]` is undefined, we assign member role to this user, so it doesn't error. 

The other || variables were added to avoid similar issues on other race conditions